packages/gatsby/src/services/initialize.ts

```diff
@@ -178,6 +179,26 @@ export async function initialize({
     })
   }
 
+  // Setup flags
+  if (config && config.flags) {
+    const flags = require(`../utils/flags`).default
+    // Get flags
+    const { validConfigFlags, message } = handleFlags(flags, config.flags)
+
+    //  set process.env for each flag
+    validConfigFlags.forEach(flag => {
+      process.env[flag.env] = `true`
+    })
+
+    //  track usage of feature
+    validConfigFlags.forEach(flag => {
+      telemetry.trackFeatureIsUsed(_.upperFirst(_.camelCase(flag.name)))
```

Comment:
Can't we add this to the config? so we don't have to this dance?

```suggestion
      telemetry.trackFeatureIsUsed(flag.telemetryId)
```

Reply:
👍